tests/js/spec/views/onboarding/configure/index.spec.jsx

```diff
@@ -162,7 +162,11 @@ describe('Configure should render correctly', function() {
         },
       });
 
-      expect(wrapper).toMatchSnapshot();
+      setTimeout(() => {
```

Comment:
the snapshot didn't change?

Reply:
Yep. Needed the timeout otherwise it'll just show a loading screen